main.js

```diff
@@ -48,21 +47,137 @@ form.addEventListener('submit', (e) => {
-//     thirdLang : 'JavaScript',
-//     fourthLang : 'Ruby'
-//   }
-// };
-// btnProject.addEventListener('click', console.log(projectData.projectLanguages));
+window.addEventListener('DOMContentLoaded', () => {
+  let getLocalStorageData = localStorage.getItem('myFormData');
+  getLocalStorageData = JSON.parse(getLocalStorageData);
+  document.getElementById('full-name').value = getLocalStorageData.fullName;
+  document.getElementById('my-email').value = getLocalStorageData.email;
+  document.getElementById('my-message').value = getLocalStorageData.message;
+
+  let closeButton = document.querySelector('.class-close');
+  closeButton.addEventListener('click', () => {
+    modal.classList.toggle('card');
+  });
+
+  let reloadPage = document.querySelectorAll('btncheck');
+  reloadPage.forEach((e) => e.addEventListener('click', () => {
+    modal.classList.toggle('card');
+  }));
+});
+
+document.addEventListener('DOMContentLoaded', () => {
+  document.getElementById('btn-submit').addEventListener('click', addDataToLocalStorage);
+});
+
+// Generating HTML content using JavaScript
+let bodyElement = document.body;
+let projectSection = document.getElementById('portfolio');
+let projectSectionTitle = document.createElement('h2');
+let underLineImage = document.createElement('img');
+let projectCard = document.createElement('div');
+let projectImage = document.createElement('img');
+let projectDetails = document.createElement('div');
+let projectHeader = document.createElement('h2');
+let projectDecription = document.createElement('p');
+let projecttools = document.createElement('div');
+let toolsunoredered = document.createElement('ul');
+let list = document.createElement('li');
+let projectButton = document.createElement('button');
+
+document.addEventListener('DOMContentLoaded', () => {
+  document.getElementById('project-image1').src = projectData[3].imag1;
+  document.getElementById('project-image2').src = projectData[3].image2;
+  document.getElementById('project-image3').src = projectData[3].image3;
+  document.getElementById('project-image4').src = projectData[3].image4;
+
+  let allbutton = document.querySelectorAll('.project-link');
+
+  allbutton.forEach((e) => e.addEventListener('click', () => {
+    modal.classList.toggle('card');
+  }));
+});
+
+projectSectionTitle.className = 'recent';
+underLineImage.className = 'line-divider2';
+projectCard.className = 'inner-element';
+
+underLineImage.src = 'img/Vector4.png';
+underLineImage.setAttribute('alt', 'Project Image');
+projectSectionTitle.innerText = 'MY RECENT PROJECT';
+projectSection.append(projectSectionTitle, underLineImage);
+
+const projectData = [
+  4,
+  {
+    projectTitle: '',
+    projectDescription: "A website created for a digital marketing agency to help improve sales and give information about company's product. This website also helps the client to contact the company for a business deal and showcases the company's different services and packages.",
+  },
+  {
```

Comment:
Buttons should stack on each other for mobile view. Please follow the design and make the pop-up on mobile the same as in the design
![mob](https://user-images.githubusercontent.com/35504149/136659437-bb1e1753-8499-49e9-aa90-c897d9dae09a.png)
.